runtime/vm/exceptiondescribe.c

```diff
@@ -343,11 +344,46 @@ iterateStackTrace(J9VMThread * vmThread, j9object_t* exception, callback_func_t
 					pruneConstructors = FALSE;
 #endif
 					romClass = findROMClassFromPC(vmThread, methodPC, &classLoader);
-					if(romClass) {
-						romMethod = findROMMethodInROMClass(vmThread, romClass, methodPC);
-						if (romMethod != NULL) {
-							methodPC -= (UDATA) J9_BYTECODE_START_FROM_ROM_METHOD(romMethod);
+					if (NULL != romClass) {
+						J9UTF8 const *utfClassName = J9ROMCLASS_CLASSNAME(romClass);
+
+						ramClass = peekClassHashTable(vmThread, classLoader, J9UTF8_DATA(utfClassName), J9UTF8_LENGTH(utfClassName));
+						if (ramClass == NULL) {
+							if (j9shr_Query_IsAddressInCache(vm, romClass, romClass->romSize)) {
+								/* Probe the application loader to determine if it has the J9Class for the current class.
+								 * This secondary probe is required as all ROMClasses from the SCC appear to be owned
+								 * by the bootstrap classloader.
+								 */
+								ramClass = peekClassHashTable(vmThread, vm->applicationClassLoader, J9UTF8_DATA(utfClassName), J9UTF8_LENGTH(utfClassName));
+								}
```

Comment:
Two braces here are overindented.

Reply:
Fixed in [force-push](https://github.com/eclipse/openj9/compare/a7aa8a4b37aed9c08825841319d2639bec3917f4..0921605921fb6482be5eb990637e36a8bf741cbc).